library/src/main/java/com/novoda/downloadmanager/notifications/NotificationDisplayer.java

```diff
@@ -239,7 +239,7 @@ private Notification buildSingleNotification(
         if (type == SynchronisedDownloadNotifier.TYPE_ACTIVE) {
             String description = batch.getDescription();
             if (TextUtils.isEmpty(description)) {
-                setSecondaryNotificationText(builder, style, remainingText);
+                setSecondaryNotificationText(builder, style, context.getString(R.string.dl__downloading));
```

Comment:
why didn't you do `context.getString(R.string.dl__remaining, remainingText)`

`<string name="dl__remaining">%s$1</string>`

so as to not break the current behaviour of all other apps? :x: